mbs_results/config.json

```diff
@@ -129,6 +135,20 @@
 
     "census_extra_calibration_group": [5043, 5113, 5123, 5203, 5233,
      5403, 5643, 5763, 5783, 5903, 6073],
-     "filter_out_questions": [11, 12 ,146]
+     "filter_out_questions": [11, 12 ,146],
+     "csw_to_spp_columns":{
```

Comment:
Has this been added as the back data is from CSW?

Reply:
You mean `csw_to_spp_columns`  dict?

Reply:
There is a mismatch with column names with csw data and spp data for an example csw has question_no while spp has questioncode , so this converts the csw column to match the spp ones, for now just these 3 need renaming